sorts.py

```diff
@@ -43,12 +43,12 @@ def chris_select_sort(values):
 
 def peter_select_sort(nums):
     newlist = []
-    while len(newlist) < nums:
-        newlist.append(min(nums))
-        nums.remove(min(nums))
-        if len(nums) == 0:
-            nums.append(newlist)
-            break
+    if len(nums) <= 1:
```

Comment:
You don't need to do any extra work for zero- or one-length inputs, the "while nums:" loop takes care of everything on its own already.